package.json

```diff
@@ -267,7 +267,7 @@
     "@types/chai": "4.1.7",
     "@types/glob": "7.1.1",
     "@types/lodash": "4.14.126",
-    "@types/mocha": "5.2.6",
+    "@types/mocha": "8.2.2",
```

Comment:
I assume no one bothered to update the typed-definitions to 9.x.x 🤔 .
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/mocha/index.d.ts

Reply:
Yeah, looks like that's what happened. I went off the latest version available on npm: https://www.npmjs.com/package/@types/mocha